src/main/java/com/ghusse/ci/violations/bbcloud/lib/CommentsProviderError.java

```diff
@@ -0,0 +1,19 @@
+package com.ghusse.ci.violations.bbcloud.lib;
+
+public class CommentsProviderError extends Error{
+  private PullRequestDescription description;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Make "description" transient or serializable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1948)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this "description" field final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1165)